### PR TITLE
Triforce Hunt GUI Bug

### PR DESCRIPTION
**Pull request by Star1468**
The latest versions of Dev/Roman's, which added variable Ganon's BK on LACS conditions, have an odd visual bug resulting from a minor oversight in Settings.py. https://discord.com/channels/274180765816848384/512048482677424138/771392222788059186
This PR fixes it by disabling the individual variable LACS conditions (medallions, stones, dungeon rewards, and tokens) in Triforce Hunt, which also disables the sliders for each of those options in the GUI.

### CODE REVIEW

**Comment by Roman971** [comment]
The "triforce" option in `shuffle_ganon_bosskey` should probably disable all these settings, instead of doing it from the triforce hunt setting, because the issue only exists because "triforce" is the default for `shuffle_ganon_bosskey` when it's disabled.

**Comment by Star1468** [comment]
Oops okay I'm still learning how to use GitHub, and am a bit of a scatterbrain. Oops, I think that fixes it now?

**Comment by Roman971** [comment]
I meant adding "triforce" in the list of disables with the corresponding settings (the same list as "remove" for example).
The "triforce" option being the default when disabled is intended, because ganon's BK is handled differently in Triforce Hunt. This option doesn't exist in the normal list of options tho, and that's why it was missed at first.

**Comment by Star1468** [comment]
Fair enough, this more like what you're thinking of?

File: SettingsList.py [hunk 1]
```diff
@@ -2884,10 +2884,10 @@ def __init__(self, name, gui_text, min, max, default, step=1,
...
             'keysanity':        {'settings': ['lacs_medallions', 'lacs_stones', 'lacs_rewards', 'lacs_tokens']},
             'lacs_vanilla':     {'settings': ['lacs_medallions', 'lacs_stones', 'lacs_rewards', 'lacs_tokens']},
-            'lacs_stones':      {'settings': ['lacs_medallions', 'lacs_rewards', 'lacs_tokens']},
-            'lacs_medallions':  {'settings': ['lacs_stones', 'lacs_rewards', 'lacs_tokens']},
-            'lacs_dungeons':    {'settings': ['lacs_medallions', 'lacs_stones', 'lacs_tokens']},
-            'lacs_tokens':      {'settings': ['lacs_medallions', 'lacs_stones', 'lacs_rewards']},
+            'lacs_stones':      {'settings': ['triforce_hunt', 'lacs_medallions', 'lacs_rewards', 'lacs_tokens']},
+            'lacs_medallions':  {'settings': ['triforce_hunt', 'lacs_stones', 'lacs_rewards', 'lacs_tokens']},
+            'lacs_dungeons':    {'settings': ['triforce_hunt', 'lacs_medallions', 'lacs_stones', 'lacs_tokens']},
+            'lacs_tokens':      {'settings': ['triforce_hunt', 'lacs_medallions', 'lacs_stones', 'lacs_rewards']},
```
Comment (Roman971) [inline SettingsList.py:2890]: It's a bit hard to explain so here is what I meant.
I haven't tested it but if disables function as I think they do, it should work.
```suggestion
            'lacs_stones':      {'settings': ['lacs_medallions', 'lacs_rewards', 'lacs_tokens']},
            'lacs_medallions':  {'settings': ['lacs_stones', 'lacs_rewards', 'lacs_tokens']},
            'lacs_dungeons':    {'settings': ['lacs_medallions', 'lacs_stones', 'lacs_tokens']},
            'lacs_tokens':      {'settings': ['lacs_medallions', 'lacs_stones', 'lacs_rewards']},
            'triforce':         {'settings': ['lacs_medallions', 'lacs_stones', 'lacs_rewards', 'lacs_tokens']},
```

**Comment by Star1468** [comment]
I just dropped that into my downloaded version of your branch- that did not work.

**Comment by Roman971** [comment]
Okay, my guess is there is a bug in the GUI code then. I can try looking at it sometime tomorrow.

**Comment by Star1468** [comment]
Probably, thanks. The first fix I tried, where I disabled LACS settings from TH, seems to work fine, but the other methods I've tried have resulted in problems. I'll defer to you on this one.

**Comment by Roman971** [comment]
The GUI bug is far from trivial to fix so we'll have to handle it manually for the time being, by disabling the sliders from the triforce hunt option like you had in the first iteration.

**Comment by Star1468** [comment]
Alright, thanks. Yeah, I figured the deeper reasons behind the bug might be harder to fix.

**Comment by dragonbane0** [comment]
Currently when a setting that disables other settings itself gets disabled, its deactivations are cleared from the list and there is another clean iteration performed to resolve any outstanding conflicts. So this is currently intentional behaviour indeed. I will look into if we can just change it, but I'm pretty sure there was some specific combination that required this setup or it would lead to settings being inaccessible when they shouldn't be. Perhaps that's no longer the case or could be resolved a different way.

**Pull request closed without merging**